README.md

```diff
@@ -18,11 +18,24 @@ To learn more about how to work with CUDA Quantum, please take a look at the
 [installation instructions][official_install] for officially released packages.
 
 If you would like to install the latest iteration under development in this
-repository and/or add your own modifications, please build CUDA Quantum from
-source following [these instructions](./Building.md).
+repository and/or add your own modifications, take a look at the [latest
+packages][github_packages] deployed on the GitHub Container Registry. For more
+information about building CUDA Quantum from source, see [these
+instructions](./Building.md).
 
 [cuda_quantum_docs]: https://nvidia.github.io/cuda-quantum/latest
 [official_install]: https://nvidia.github.io/cuda-quantum/latest/install.html
+[github_packages]:
+    https://github.com/orgs/NVIDIA/packages?repo_name=cuda-quantum
+
+<img
+align="right"
+src="https://github.com/NVIDIA/cuda-quantum/actions/workflows/deployments.yml/badge.svg?event=workflow_dispatch&branch=main"
+/>
+<img
+align="right"
+src="https://github.com/NVIDIA/cuda-quantum/actions/workflows/publish_docs.yml/badge.svg?branch=main"
+/><br/>
```

Comment:
I think it is aesthetically pleasing to put the badges in the top left of the README. It might also make easier to identify a problem if our README gets bigger.

Reply:
done